src/pages/components/variants/inputCheckbox/Disabled.tsx

```diff
@@ -1,16 +1,26 @@
 import { InputCheckbox } from '@boston-scientific/anatomy-react';
+import { Fieldset } from '@boston-scientific/anatomy-react';
 import Example from 'shared/components/Example';
 
 const Disabled = (): JSX.Element => {
   return (
-    <Example>
-      <div className="bsds-form-control">
-        <InputCheckbox label="Disabled checkbox" disabled />
-      </div>
-      <div className="bsds-form-control">
-        <InputCheckbox label="Disabled checked checkbox" disabled defaultChecked />
-      </div>
-    </Example>
+    <>
+      <Example>
+        <div className="bsds-form-control">
+          <InputCheckbox label="Disabled checkbox" disabled />
+        </div>
+        <div className="bsds-form-control">
+          <InputCheckbox label="Disabled checked checkbox" disabled defaultChecked />
+        </div>
```

Comment:
wonder if we still need this now that we show group with one checked right below since none of the other non-group checkbox previews have an unchecked + checked example in it